### ISSUE TITLE
Content in “Copilot Chat Section” is not visible after applying Resize in 'Open source AI code editor' screen: A11y_ Visual Studio Code Services_Open Source AI code editor screen _Resize.

### ISSUE THREAD

**Issue by AnKushSingh05** [microsoft/vscode-docs#9308]
### GitHub Tags: 

#A11yTCS; #A11yMAS; #A11ySev2; #AI Limited; #DesktopWeb; #ChromiumEdge; #Win11; #Visual Studio Code Services; #E2E_Visual Studio Code Services_Web_Jan26; #WCAG1.4.4; #Reflow; #ES Chat;  

"Please do not close this bug. This bug should only be closed by TCS, C+AI Accessibility tester after bug verification." 

Try [ES Chat](https://nam06.safelinks.protection.outlook.com/?url=https:%2f%2feng.ms%2fdocs%2fcoreai%2fdevdiv%2fone-engineering-system-1es%2f1es-docs%2fes-chat%2findex&data=05%7c02%7cv-pawanksing%40microsoft.com%7c26cff27270c448f48a0e08de3f569146%7c72f988bf86f141af91ab2d7cd011db47%7c1%7c0%7c639017838221175207%7cUnknown%7cTWFpbGZsb3d8eyJFbXB0eU1hcGkiOnRydWUsIlYiOiIwLjAuMDAwMCIsIlAiOiJXaW4zMiIsIkFOIjoiTWFpbCIsIldUIjoyfQ%3D%3D%7c0%7c%7c%7c&sdata=ju8Raw6mIhT5G3gWbZSkS0CAN2zkv2ycA5ifSSqoxOw%3D&reserved=0) to learn more about the MAS rule and how to fix the issue. If you need more help, use our [Teams channel](https://nam06.safelinks.protection.outlook.com/?url=https:%2f%2fteams.microsoft.com%2fl%2fchannel%2f19%253ADeCtYeREyQHw5jYsz9PPc3hpVw48m-tOlTkQ1jEn0eM1%2540thread.tacv2%2fCAI%2520Accessibility%2520Support%2520Channel?groupId%3Df5eb428c-9e6e-4c16-8143-33fe3961a9ae%26tenantId%3D72f988bf-86f1-41af-91ab-2d7cd011db47&data=05%7c02%7cv-pawanksing%40microsoft.com%7c26cff27270c448f48a0e08de3f569146%7c72f988bf86f141af91ab2d7cd011db47%7c1%7c0%7c639017838221198042%7cUnknown%7cTWFpbGZsb3d8eyJFbXB0eU1hcGkiOnRydWUsIlYiOiIwLjAuMDAwMCIsIlAiOiJXaW4zMiIsIkFOIjoiTWFpbCIsIldUIjoyfQ%3D%3D%7c0%7c%7c%7c&sdata=DCNVsZe8oDZWqm97AI0ee4OoWWecR6zw6qlhTuzP7MI%3D&reserved=0) or [office hours](https://nam06.safelinks.protection.outlook.com/?url=https:%2f%2foutlook.office.com%2fbook%2fmicrosoft2779007%2540microsoft.onmicrosoft.com%2fs%2fbLeGBltMW0qI0WRLsTqx2A2?ismsaljsauthenabled&data=05%7c02%7cv-pawanksing%40microsoft.com%7c26cff27270c448f48a0e08de3f569146%7c72f988bf86f141af91ab2d7cd011db47%7c1%7c0%7c639017838221214666%7cUnknown%7cTWFpbGZsb3d8eyJFbXB0eU1hcGkiOnRydWUsIlYiOiIwLjAuMDAwMCIsIlAiOiJXaW4zMiIsIkFOIjoiTWFpbCIsIldUIjoyfQ%3D%3D%7c0%7c%7c%7c&sdata=CnCCdDgEmll2oUQUdMjXusAgV8PneqZVYvzgxAfk7Ik%3D&reserved=0). 

### Environment Details: 

Application Name: Visual Studio Code Services 
URL: [https://code.visualstudio.com](https://nam06.safelinks.protection.outlook.com/?url=https%3A%2F%2Fcode.visualstudio.com%2F&data=05%7C02%7Cv-kduvvuri%40microsoft.com%7Cf492af0a8c204ceb833c08de5d7335b8%7C72f988bf86f141af91ab2d7cd011db47%7C1%7C0%7C639050947034495318%7CUnknown%7CTWFpbGZsb3d8eyJFbXB0eU1hcGkiOnRydWUsIlYiOiIwLjAuMDAwMCIsIlAiOiJXaW4zMiIsIkFOIjoiTWFpbCIsIldUIjoyfQ%3D%3D%7C0%7C%7C%7C&sdata=pGQTzxzKOp7hs0ceHAv0XoOcNW3%2Byeh5%2F%2FMFLbSZonA%3D&reserved=0)  
Chromium Edge: Version 144.0.3719.104 (Official build) dev (64-bit) 
Edition: Windows 11 Enterprise Insider Preview 
Version: 24H2 
OS build: 26558.1000 

**Pre Requisites:** 

1. Change the display resolution to 1366*768. 
2. Change the text size to 200 %. 
 
### Repro steps: 

1. Hit the URL: [https://code.visualstudio.com](https://nam06.safelinks.protection.outlook.com/?url=https%3A%2F%2Fcode.visualstudio.com%2F&data=05%7C02%7Cv-kduvvuri%40microsoft.com%7Cf492af0a8c204ceb833c08de5d7335b8%7C72f988bf86f141af91ab2d7cd011db47%7C1%7C0%7C639050947034495318%7CUnknown%7CTWFpbGZsb3d8eyJFbXB0eU1hcGkiOnRydWUsIlYiOiIwLjAuMDAwMCIsIlAiOiJXaW4zMiIsIkFOIjoiTWFpbCIsIldUIjoyfQ%3D%3D%7C0%7C%7C%7C&sdata=pGQTzxzKOp7hs0ceHAv0XoOcNW3%2Byeh5%2F%2FMFLbSZonA%3D&reserved=0) 
2. Tab navigate to Agent Mode image. 
3. Tab navigate to “Copilot Chat Section”  
4. Observe that Content in “Copilot Chat Section” is not visible after applying Resize in 'Open source AI code editor' screen. 

### Actual Result: 

When text resizing is applied (e.g., zooming in up to 200%), the content in the "Copilot Chat Section" is not visible in the "Open Source AI Code Editor" screen. This results in users being unable to access or read the chat content. 

### Expected Result: 

The "Copilot Chat Section" should remain visible and accessible when text resizing is applied. The layout should adapt dynamically to ensure that all content is displayed without loss of functionality or requiring excessive horizontal scrolling. 

### User Impact: 

Low vision users may encounter significant challenges if the content within the 'Copilot Chat Section' becomes inaccessible after resizing the 'Open Source AI Code Editor' screen. Ensuring that all content remains visible and accessible during resizing is essential to support users with low vision and provide an inclusive experience. 

### Attachment:

https://github.com/user-attachments/assets/74e552fe-79d9-41e2-a175-6068dc950844

**Comment by rzhao271** [comment]
@hawkticehurst the main issue here is the Agent Mode demo has cut-off content when rendered on a phone at 200% zoom or on a 1366x768 window at 200% zoom.

**Comment by hawkticehurst** [comment]
Hi @AnKushSingh05! Transitioning from having a scrollable container on desktop to a non-scrollable container on mobile is an intentional decision. We decided that it would be better to do this so we avoid "trapping" scroll on mobile touch devices because otherwise the scrollable container would take up most of the width on mobile devices and be place where people would like get stuck while scrolling on their mobile devices.

This tradeoff is okay because these are supposed to represent visual graphics on the page similar to images. They just happen to be implemented using HTML and CSS to make it easier to maintain and update these graphics over time. It's intented for this (and other) graphics on the homepage to be consumed as a "whole" or a big "single unit", meaning it doesn't matter if the entirety of content inside the copilot chat section is visible on desktop and not visible on mobile. The entire graphic is just supposed to indicate and support the advertisement that VS Code has agentic coding capabilities.

**Comment by hawkticehurst** [comment]
Fixed! We made all graphics inert and fully non-interactive + added role=image / aria-labels to the direct parent element of all graphics. Should be deployed to the website within the next few days

**Comment by hawkticehurst** [comment]
Reopened for testing

**Comment by kapilvaishna** [comment]
Issue is not repro so closing the bug.
#Closed; #A11yFixed; #Regressed:04-16-2026;

https://github.com/user-attachments/assets/aa2c4d99-f873-4ac9-ac05-e8582b077f7c
